deepchem/feat/huggingface_featurizer.py

```diff
@@ -0,0 +1,25 @@
+from deepchem.feat.base_classes import Featurizer
+
+
+class HuggingFaceFeaturizer(Featurizer):
```

Comment:
This looks useful. Having the ability to use arbitrary huggingface tokenizers will be useful for DeepChem model building and chemberta